data/json/monsters/mechsuits.json

```diff
@@ -0,0 +1,139 @@
+    "weight": 700388,
+    "hp": 320,
+    "speed": 120,
+    "material": [ "superalloy" ],
+    "symbol": "M",
+    "color": "blue",
+    "aggression": -50,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 3,
+    "melee_dice_sides": 8,
+    "melee_cut": 5,
+    "armor_bash": 15,
+    "armor_cut": 25,
+    "vision_night": 100,
+    "mech_str_bonus": 5,
+    "mech_battery": "huge_atomic_battery_cell",
+    "mech_weapon": "recon_mech_laser",
+    "revert_to_itype": "broken_mech_recon",
+    "death_function": [ "BROKEN" ],
+    "death_drops": { "groups": [ [ "robots", 20 ] ] },
+    "flags": [
+      "SEES",
+      "MECH_RECON_VISION",
+      "HEARS",
+      "MILITARY_MECH",
+      "BASHES",
+      "PUSH_MON",
+      "ELECTRONIC",
+      "NOHEAD",
+      "NO_BREATHE",
+      "RIDEABLE_MECH",
+      "PRIORITIZE_TARGETS",
+      "PATH_AVOID_DANGER_2"
+    ]
+  },
+  {
+    "id": "mon_mech_combat",
+    "type": "MONSTER",
+    "name": "X-02: 'Grunt' Combat Mech",
+    "description": "The Boeing-Daewoo CMES ( Combat Mechanical Exoskeleton Suit ), a recent acquisition by the US military, it was designed to be used in a fire support role, due to its fearsome integrated gatling laser.  It was not deployed before the Cataclsym hit, though there were a few prototypes in the field.  You may be able to hack it to accept you as its pilot.  Like all mech-suits it can act as a UPS from its large battery.",
+    "default_faction": "mech_bot",
+    "species": [ "ROBOT" ],
+    "diff": 5,
+    "volume": "1480388 ml",
+    "weight": 1480388,
+    "hp": 550,
+    "speed": 65,
+    "material": [ "superalloy" ],
+    "symbol": "M",
+    "color": "red",
+    "aggression": -50,
+    "morale": 100,
+    "melee_skill": 2,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_cut": 0,
+    "armor_bash": 25,
+    "armor_cut": 45,
+    "vision_night": 50,
+    "mech_str_bonus": 10,
+    "mech_battery": "huge_atomic_battery_cell",
+    "mech_weapon": "gatling_mech_laser",
+    "revert_to_itype": "broken_mech_combat",
+    "death_function": [ "BROKEN" ],
+    "death_drops": { "groups": [ [ "robots", 20 ] ] },
+    "flags": [
+      "SEES",
+      "HEARS",
+      "MILITARY_MECH",
+      "BASHES",
+      "PUSH_MON",
+      "ELECTRONIC",
+      "NOHEAD",
+      "NO_BREATHE",
+      "RIDEABLE_MECH",
+      "PRIORITIZE_TARGETS",
+      "PATH_AVOID_DANGER_2",
+      "LOUDMOVES"
+    ]
+  },
+  {
+    "id": "mon_mech_lifter",
+    "type": "MONSTER",
+    "name": "X-01:'Jack' Lifting Mech",
+    "description": "The Boeing-Daewoo LMES ( Lifting Mechanical Exoskeleton Suit ), a recent acquisition by the US military, it was designed to be piloted by an operator to load and unload heavy material, and for limited combat support roles, it was not deployed before the Cataclsym hit, though there were a few prototypes in the field.  You may be able to hack it to accept you as its pilot.  Like all mech-suits it can act as a UPS from its large battery.",
```

Comment:
```suggestion
    "description": "The Boeing-Daewoo LMES (Lifting Mechanical Exoskeleton Suit), a recent acquisition by the US military, it was designed to be piloted by an operator to load and unload heavy material, and for limited combat support roles, it was not deployed before the Cataclysm hit, though there were a few prototypes in the field.  You may be able to hack it to accept you as its pilot.  Like all mech-suits it can act as a UPS from its large battery.",
```